.pipelines/java_packaging.yml

```diff
@@ -25,7 +25,7 @@ stages:
 
     - script: |
         cd $(Build.ArtifactStagingDirectory)
-        jar xvf onnxruntime-extensions-0.11.0.jar
+        jar xvf onnxruntime-extensions-$(cat version.txt).jar
```

Comment:
cat and $() work on Windows?

Reply:
cat works only in PowerShell script, $() works regardless.